gnoi_infra_deps.bzl

```diff
@@ -0,0 +1,53 @@
+# Copyright 2021 Google LLC
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Dependencies to build PINS infra."""
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+def gnoi_infra_deps():
+    """Declare the third-party dependencies necessary to build GNOI infrastructure"""
+
+    if not native.existing_rule("com_github_grpc_grpc"):
+       http_archive(
+            name = "com_github_grpc_grpc",
+            # Move to newer commit to avoid the use of com_github_google_re2.
```

Comment:
Can we remove this comment and instead use the latest release? (https://github.com/grpc/grpc/releases/tag/v1.37.1)

Reply:
Done